### PR TITLE
Chore!: remove not needed apis and summary vc support

### PR DESCRIPTION
**Pull request by nitin-vavdiya**
### What
1. Issue BPN credentials API is removed 
2. Issue Membership credential API is removed
3. Issue Framework credential API is removed
4. Summary credential support is removed
5. Test cases modification 

### Why
In the new release of tractus-X , we have a new Issuer component and all VC formation(VS JSON structure) will be taken care by this component

The responsibility of MIW is now to sign VC provided by the issuer component only. 

EDC is using IATP protocol instead of summary VC
 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs/api/openapi_v001.json
Comment (borisuu) [inline docs/api/openapi_v001.json:1]: Why is this file changed?

**Review by borisuu — CHANGES_REQUESTED** [review]

File: docs/api/openapi_v001.json
Comment (borisrizov-zf) [inline docs/api/openapi_v001.json:1]: Why is this file changed?

**Review by borisrizov-zf — CHANGES_REQUESTED** [review]

File: docs/api/openapi_v001.json
Comment (nitin-vavdiya) [reply docs/api/openapi_v001.json:1]: We removed 3 APIs and DTOs so I thought we also need to remove API specification

Comment (borisrizov-zf) [reply docs/api/openapi_v001.json:1]: True, but the formatting seems off, too many lines are changed. Could you check that.

Comment (nitin-vavdiya) [reply docs/api/openapi_v001.json:1]: fixed with : https://github.com/eclipse-tractusx/managed-identity-wallet/pull/315/commits/b10fa38eeae221b48adc6ebdace8ffdf4781bb09

**Review by borisrizov-zf — APPROVED** [review]

**Pull request merged**